### PR TITLE
Format /api code samples

### PR DESCRIPTION
**Pull request by razor-x**
Runs a code formatter for each code sample. I think we may need to adjust the max line length and possibly force line breaks, but we can tweak the formatting behavior now that we have this in place.

I do not feel strongly about any of the chosen formatters, I just picked ones that were popular and worked. If one of the formatters I chose here needs to be swapped out to meet our needs, it should be straight forward.

### CODE REVIEW

File: docs/api/acs/systems/list.md [hunk 1]
```diff
@@ -36,24 +40,30 @@ response includes all access control systems connected to your workspace.
     {% tab title="Ruby" %}
   ### Request
   ```ruby
-  seam.acs.systems.list(connected_account_id: "8d7e0b3a-b889-49a7-9164-4b71a0506a33")
+  ====================
```
Comment (DebbieAtSeam) [inline docs/api/acs/systems/list.md:43]: @razor-x What is this line for?

File: docs/api/acs/systems/list.md [hunk 2]
```diff
@@ -36,24 +40,30 @@ response includes all access control systems connected to your workspace.
...
+seam.acs.systems.list(connected_account_id: "8d7e0b3a-b889-49a7-9164-4b71a0506a33")
   ```
 
   ### Response
   ```ruby
-  [<Seam::AcsSystems:0x00000
-acs_system_id="8d7e0b3a-b889-49a7-9164-4b71a0506a33">]
+  ====================
```
Comment (DebbieAtSeam) [inline docs/api/acs/systems/list.md:49]: Same here

File: docs/api/acs/systems/list.md [hunk 3]
```diff
@@ -14,12 +14,16 @@ response includes all access control systems connected to your workspace.
     {% tab title="JavaScript" %}
   ### Request
   ```javascript
-  await seam.acs.systems.list({"connected_account_id":"8d7e0b3a-b889-49a7-9164-4b71a0506a33"})
+  await seam.acs.systems.list({
+  connected_account_id: "8d7e0b3a-b889-49a7-9164-4b71a0506a33",
+});
+
```
Comment (DebbieAtSeam) [inline docs/api/acs/systems/list.md:20]: @razor-x Why is an extra, blank line added at the end of each code block?

Comment (DebbieAtSeam) [reply docs/api/acs/systems/list.md:20]: Also, could we please run the resulting generated code samples by Dawn? She had opinions on how to format/indent/etc. the code samples. You can see her suggestions implemented in the ACS code samples in the current docs.

**Comment by razor-x** [comment]
@DebbieAtSeam Realized we should add some more code samples since only one page has them right now: https://github.com/seamapi/docs/issues/324

File: docs/api/acs/systems/list.md (hunk 3 above)
Comment (razor-x) [reply docs/api/acs/systems/list.md:20]: Sure. This PR just adds the formatters to the pipeline so we get basic validation that code samples are valid and tweaking the rules can come next 👍🏻

**Pull request merged**